### ISSUE TITLE
Inheritance and Freqeuncy in HpoDisease

### ISSUE THREAD

**Issue by pnrobinson** [Phenomics/ontolib#30]
Hi!
Currentl, the HPO Disease models almost always have just one mode of inheritance, and in the few cases where there are two (they should be split imho) we never have frequency data. 
I think that having a list with a simple Termid would be more realistic.
See HpoDisease line 38:
```
List<TermIdWithFrequency> modesOfInheritance
````
But do you see a reason to user TermidWithFrequency here?

**Comment by holtgrewe** [comment]
That's the way it was in @drseb's original library.

How would we handle issues here? If HPO guarantees to only have one mode of inheritance (by splitting), we can remove the list here. However, I would rather keep the frequency field and put "unknown" (`null`?)

**Comment by pnrobinson** [comment]
What is the use case for Frequency data with the inheritance field? Even if there was a use case, we currently have no data at all on the frequency of different modes of inheritance of the corresponding diseases (it is a pretty rare case anyway).
It will not hurt to keep the field and ignore it, but I think it is better to have the data structure reflect the data...

**Comment by iamkoehler** [comment]
> That's the way it was in @drseb's original library.

Yes, I do not treat annotations (i.e. a link between "something" and an ontology class) differently, depending on which sub ontology the ontology class belongs to. This is happening downstream. I think Peter is right, but it would be good to check how often exactly we have multiple different inheritance annotations to one disease in HPOA, before making this change.
